modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ConfigurationContainer.tsx

```diff
@@ -25,7 +27,11 @@ export function ConfigurationContainer({
 	return (
 		<div className="lfr-objects__object-definition-details-configuration">
 			<Toggle
-				disabled={isReadOnly || !hasUpdateObjectDefinitionPermission}
+				disabled={
+					isReadOnly ||
```

Comment:
@igor-franca @alinedoleron I think it's better to create a const for this verification, since we're using it many times.